templates/mimic.html

```diff
@@ -22,7 +22,7 @@ <h1 class="site-section-title" id="MIMIC">MIMIC</h1>
 
       <h2 class="site-section-meta border-bottom" id="Overview">Overview</h2>
       <p>
-        MIMIC is a publicly available dataset developed by the Laboratory for Computational Physiology that comprises deidentified health data associated with thousands of intensive care unit admissions. The dataset is widely used by investigators and engineers around the world, helping to drive research in clinical informatics, epidemiology, and machine learning.
+        <a href="https://mimic.physionet.org" target="_blank">MIMIC</a></li> is a publicly available dataset developed by the Laboratory for Computational Physiology that comprises deidentified health data associated with thousands of intensive care unit admissions. The dataset is widely used by investigators and engineers around the world, helping to drive research in clinical informatics, epidemiology, and machine learning.
```

Comment:
I don't think the `</li>` is intended to be here.

Reply:
Ahhh nope! Good catch! Didn't show up in any obvious way without looking harder at the code!